fbw-a32nx/src/systems/fmgc/src/flightplanning/new/plans/FlightPlan.ts

```diff
@@ -471,7 +471,7 @@ export class FlightPlan<P extends FlightPlanPerformanceData = FlightPlanPerforma
      *
      * The union type in the signature is to work around https://github.com/microsoft/TypeScript/issues/28662.
      */
-    setPerformanceData<k extends(keyof (P & FlightPlanPerformanceDataProperties)) & string>(key: k, value: P[k], notify = true) {
+    setPerformanceData<k extends(keyof (P & FlightPlanPerformanceDataProperties)) & string>(key: k, value: P[k] | null, notify = true) {
```

Comment:
Why can value be `null`? If anything, it should be undefined I think?

Reply:
I always prefer null, because it has to be explicit, and more importantly, it avoids bugs with the EventBus.

Reply:
I agree, but then the performance data properties should also be marked as nullable.

Reply:
I removed this part, and marked the values themselves as nullable.